mozilla/blurts-server
# Report CSP violations to Sentry

# References:

Jira: MNTOR-3345
Figma: N/A



# Description

This follows the instructions from https://docs.sentry.io/security-legal-pii/security/security-policy-reporting/#content-security-policy, to report CSP violations to Sentry.

# How to test

I'm not sure; I can see the HTTP response headers, but I don't see them show up in Sentry. I added the following to a page, which gets shown as a CSP violation in the devtools:

```
        <div dangerouslySetInnerHTML={{__html: "<script src='https://example.com/script.js'></script>"}}></div>
```

But I'm not sure whether I should be seeing the reporting somewhere, and where they should show up in Sentry. Possibly I'm looking in the wrong place?
